src/domain/faqs/FAQs.module.scss

```diff
@@ -60,7 +60,7 @@
         text-align: center;
     }
     & p > span{
```

Comment:
`& p` is the same as just  `p` in this example ( I know this wasn't you but they still annoy me - there are a few below if you care enough)

Reply:
```suggestion
    p > span{
```

Reply:
Probably best to do this in a separate change.